variables.md

```diff
@@ -179,3 +179,4 @@ ZDUPREPOS | string | | Comma separated list of repositories to be added/used for
 ZFCP_ADAPTERS | string | | Comma separated list of available ZFCP adapters in the machine (usually 0.0.fa00 and/or 0.0.fc00)
 LINUXRC_BOOT | boolean | true | To be used only in scenarios where we are booting an installed system from the installer medium (for example, a DVD) with the menu option "Boot Linux System" (not "boot From Hard Disk"). This option uses linuxrc.
 ZYPPER_WHITELISTED_ORPHANS | string | empty | Whitelist expected orphaned packages, do not fail if any are found. Upgrade scenarios are expecting orphans by default. Used by console/orphaned_packages_check.pm
+PUBLIC_CLOUD_CONTAINER_IMAGES_REPO | string | | The Container images repository where the container image are stored
```

Comment:
Just curious, does this repo somehow be created in advance?

Reply:
Yes, I did and it should be created in advance, but I will investigate if there is a way to create the repo directly during the test execution.

Reply:
What does this repo mean?

Reply:
This is the container repository in the public cloud where the images will be uploaded

Reply:
```suggestion
PUBLIC_CLOUD_CONTAINER_IMAGES_REPO | string | | The Container images repository in Public Cloud
```

I think it is obvious that Container Images Repo is used to store container images :)

Reply:
done